config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
+AUTH_TESSEL_OA2="https://client_key@auth.tessel.io"
```

Comment:
I was piecing this together from all the keys sprinkled throughout the readme. Is `AUTH_TESSEL_OA2=http://tessel-proxy-dev:the-tessel-devenv@localhost:3002` all they need? I thought they would want to use their own api keys.